### DISCUSSION TITLE
Official Docker image

### DISCUSSION THREAD

**Discussion by JCHacking** [conductor-oss/conductor#192] [Ideas]
I have seen this idea started, but it has not been maintained. Is there any plan to have an official driver image in DockerHub?

**Comment by jgard** [comment]
Came here to ask the same.  Wondering if @v1r3n has any context, as [they pushed the 3.15.0 one that is out there](https://hub.docker.com/r/conductoross/conductor-standalone/tags).

**Comment by dradoaica** [comment]
They pushed another one 14 days ago: https://hub.docker.com/layers/conductoross/conductor/3.21.24-rc.1!

**Comment by v1r3n** [comment]
`3.22.0` should be out this week.

**Comment by v1r3n** [comment]
### Latest published version
`conductoross/conductor:latest` 

### pinned to a specific version (Recommended)
`conductoross/conductor:VERSION`
